custom_components/e3dc_rscp/sensor.py

```diff
@@ -385,6 +385,40 @@ async def async_setup_entry(
         E3DCSensor(coordinator, description, entry.unique_id)
         for description in SENSOR_DESCRIPTIONS
     ]
+
+    # Add Sensor descriptions for additional powermeters
+    for powermeter_config in coordinator.get_e3dcconfig()["powermeters"]:
+        if powermeter_config["index"] != 0:
+            energy_description = SensorEntityDescription(
+                has_entity_name=True,
+                name=powermeter_config["name"] + " - total",
+                key=powermeter_config["key"] + "_total",
+                translation_key=powermeter_config["key"] + "_total",
```

Comment:
Can you please add the neccessary translation keys to the string files as well?

Reply:
Done; Please note that i am not strictly consistent when the name includes "powermeter". I did so deliberately per case. When based on the powermeter's name it is clear that it is a PM, i skipped "powermeter" to keep the names short but added it in those cases where it may lead to abiguity/confusion. (e.g. "unused" leads to unused powermeter" but "additional consumption" stays "additional consumption").
Furthermore, i changed the underscores in the keys to dashes for consistency with your other keys.